src/config/spawn.ts

```diff
@@ -62,6 +81,7 @@ export const spawnLocal: T.SpawnConductorFn = async (player: Player, { handleHoo
       player,
       name,
       kill: async (...args) => {
+        lairHandle.kill()
         // wait for it to be finished off before resolving
         const killPromise = new Promise((resolve) => {
```

Comment:
might want to make the killPromise a Promise.all, and set up two promises in the array, one for each of the child process handles

Reply:
yah, good idea